server/plugins/stencil-editor/stencil-editor.module.js

```diff
@@ -30,6 +31,7 @@ module.exports.register = function (server, options, next) {
     options.themeServer.ext('onRequest', handlers.onRequest);
 
     // When using stencil-cli variationId = configurationId
+    configurationId = themeConfig.variationIndex + 2;
```

Comment:
the + 1 thig is only a convention. Because indexes start with 0 but ids start with 1. So an id `1` is index `0`